.github/workflows/go.yml

```diff
@@ -22,9 +22,19 @@ jobs:
       run: make test
 
     - name: Coverage
-      run: make test-create-coverage
+      run: |
+        make test-create-coverage
+        COVERAGE=$(go tool cover --func=cover.out |  grep total | grep -Eo '[0-9]+\.[0-9]+')
+        echo "testCoverage=${COVERAGE}" >> $GITHUB_ENV
 
     - uses: actions/upload-artifact@v2
       with:
         name: test-coverage
         path: coverage.html
+
+    - uses: ouzi-dev/commit-status-updater@v1.1.0
```

Comment:
build fails with:
` Error: error while setting context status: Resource not accessible by integration`

Reply:
yes, yes, I was aware of that, I need to add a credential ;-)